tests/helpers/test_entity_component.py

```diff
@@ -93,67 +91,78 @@
     platform1_setup = Mock(side_effect=Exception("Broken"))
     platform2_setup = Mock(return_value=None)
 
-    mock_entity_platform(hass, "test_domain.mod1", MockPlatform(platform1_setup))
-    mock_entity_platform(hass, "test_domain.mod2", MockPlatform(platform2_setup))
+    mock_entity_platform(hass, "test_domain.mod1",
+                         MockPlatform(platform1_setup))
+    mock_entity_platform(hass, "test_domain.mod2",
+                         MockPlatform(platform2_setup))
 
     component = EntityComponent(_LOGGER, DOMAIN, hass)
 
     assert not platform1_setup.called
     assert not platform2_setup.called
 
     component.setup(
-        OrderedDict(
-            [
-                (DOMAIN, {"platform": "mod1"}),
-                ("{} 2".format(DOMAIN), {"platform": "non_exist"}),
-                ("{} 3".format(DOMAIN), {"platform": "mod2"}),
-            ]
-        )
-    )
+        OrderedDict([
+            (DOMAIN, {
+                "platform": "mod1"
+            }),
+            ("{} 2".format(DOMAIN), {
+                "platform": "non_exist"
+            }),
+            ("{} 3".format(DOMAIN), {
+                "platform": "mod2"
+            }),
+        ]))
 
     await hass.async_block_till_done()
     assert platform1_setup.called
     assert platform2_setup.called
 
 
 @asynctest.patch(
-    "homeassistant.helpers.entity_component.EntityComponent" ".async_setup_platform",
+    "homeassistant.helpers.entity_component.EntityComponent"
+    ".async_setup_platform",
     return_value=mock_coro(),
 )
-@asynctest.patch(
-    "homeassistant.setup.async_setup_component", return_value=mock_coro(True)
-)
+@asynctest.patch("homeassistant.setup.async_setup_component",
+                 return_value=mock_coro(True))
 async def test_setup_does_discovery(mock_setup_component, mock_setup, hass):
     """Test setup for discovery."""
     component = EntityComponent(_LOGGER, DOMAIN, hass)
 
     component.setup({})
 
-    discovery.load_platform(
-        hass, DOMAIN, "platform_test", {"msg": "discovery_info"}, {DOMAIN: {}}
-    )
+    discovery.load_platform(hass, DOMAIN, "platform_test",
+                            {"msg": "discovery_info"}, {DOMAIN: {}})
 
     await hass.async_block_till_done()
 
     assert mock_setup.called
-    assert ("platform_test", {}, {"msg": "discovery_info"}) == mock_setup.call_args[0]
+    assert ("platform_test", {}, {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)